services/ui-src/src/utils/tables/mapNaaarStandardsData.ts

```diff
@@ -20,7 +20,7 @@ export const mapNaaarStandardEntity = <T>(
     if (key === "standard_coreProviderType") {
       const providerKey = parentObj[0].key;
       const providerId = providerKey.split("standard_coreProviderType-").pop();
-      otherText = entity[`${key}-${providerId}-otherText`];
+      otherText = entity[`${key}-${providerId}`];
```

Comment:
will this work on old reports or just new ones? I tried to test locally but wasn't showing up on the new one for me either :/